src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -58,6 +77,31 @@ public EditCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_ADDRESS).isPresent()) {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
+        if (argMultimap.getValue(PREFIX_NRIC).isPresent()) {
+            editPersonDescriptor.setNric(ParserUtil.parseNric(argMultimap.getValue(PREFIX_NRIC).get()));
+        }
+        if (argMultimap.getValue(PREFIX_LICENCE_PLATE).isPresent()) {
+            editPersonDescriptor.setLicencePlate(
+                    ParserUtil.parseLicencePlate(argMultimap.getValue(PREFIX_LICENCE_PLATE).get()));
+        }
+
+        //        if (argMultimap.getValue(PREFIX_COMPANY).isPresent()) {
+        //            editPersonDescriptor.setCompany(
+        //            ParserUtil.parseCompany(argMultimap.getValue(PREFIX_COMPANY).get()));
+        //        }
+
+        if (argMultimap.getValue(PREFIX_POLICY_NUMBER).isPresent()) {
```

Comment:
I'm worried about the case where the Person originally has no policy (i.e. NOPOLICY, 01-01-1000, 01-01-1000), but the edits are only made such that the PolicyNumber is edited but the other fields are untouched (i.e. edit pn/ABC123). Does the parser catch this case? Not sure if this is a cause for concern